portfolio/src/main/webapp/contactList.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Personal Portfolio</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Serif&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet">
+    
+    
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div  class="mainIntro">
+      <div class="navbar">
+        <img src="images/logo.png" class="logo" alt="myLogo" onclick = "window.location.assign(contactList.html)">
+        <nav>
+          <ul id="menu">
+            <li><a href="index.html" id="home">Home</a></li>
+            <li><a href="about.html" id="about">About</a></li>
+            <li><a href="projects.html" id="project">Projects</a></li>
+            <li><a href="contact.html" id="contact">Contact</a></li>
+          </ul>
+        </nav>
+        <img src="images/menuLogo.png" class="menuLogo" onclick="toggleMenu()" alt="bars">
+      </div>
+
+      <div class="myInfo">
+         <ul id="listOfContacts"></ul>
+        
+    </div>
+  </div>
+
+  <div class="connectMe">
+      <img src="images/github.png" onclick="window.location.assign('https://www.github.com/S3nnah39')" alt="githubLogo">
+
+      <img src="images/linkedin.png" onclick="window.location.assign('https://www.linkedin.com/in/tasha-ogoti')" alt="linkedinLogo">
+  </div>
+
+<!--
+    <div id="content">
+      <p>Click <a href="https://mapmonkey.s3nnah39.repl.co/index.html" target="_blank">here</a> to see content a website I have ever created.</p>
+      <p>Click here to get a random message:</p>
+      <button onclick="addRandomGreeting()" id="helloBtn">Hello</button>
+      <div id="greeting-container"></div>
+    </div>
+-->
```

Comment:
If this isn't needed anymore, let's remove it so that our files aren't cluttered with unused code.